ttnn/cpp/ttnn/operations/moreh/moreh_linear/moreh_linear_pybind.cpp

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_linear_pybind.hpp"
+
+#include "pybind11/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_linear/moreh_linear.hpp"
+
+namespace ttnn::operations::moreh::moreh_linear {
+void bind_moreh_linear_operation(py::module& module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_linear,
+        "Moreh Linear Operation",
+        ttnn::pybind_arguments_t{
+            py::arg("input"),
+            py::arg("weight"),
+            py::kw_only(),
+            py::arg("bias") = std::nullopt,
+            py::arg("output") = std::nullopt,
+
```

Comment:
No need of space here

Reply:
resolved in [3f363ae](https://github.com/tenstorrent/tt-metal/pull/12812/commits/3f363ae0ef41cd058967757ab165f68c13da2d26)